src/views/viewer/ChannelView/ChannelAbout.styles.ts

```diff
@@ -37,14 +36,18 @@ export const Details = styled.div`
   margin-bottom: ${sizes(4)};
 `
 
+export const DetailsMember = styled(Details)`
+  display: flex;
```

Comment:
You should add `align-items: center` here
![Zrzut ekranu 2022-01-11 o 17 13 46](https://user-images.githubusercontent.com/13706246/148979651-8e6b580b-5fd9-43e6-be7d-b57b48ab594a.png)
![Zrzut ekranu 2022-01-11 o 17 13 53](https://user-images.githubusercontent.com/13706246/148979656-1fdf0284-8d89-491a-9877-796b4406ab31.png)